config/i18n/LazyImportPlugin.ts

```diff
@@ -0,0 +1,23 @@
+import { BackendModule } from 'i18next';
+
+const LazyImportPlugin: BackendModule = {
+  type: 'backend',
+  init: function (services, backendOptions, i18nextOptions) {
+  },
+  read: function (language, namespace, callback) {
+    import(/* webpackChunkName: "translations/[request]" */ `../../locales/${language}/${namespace}.json`).then(
+      (obj) => {
+        callback(null, obj);
+      }
+    );
+  },
+
+  save: function (language, namespace, data) {
```

Comment:
it looks like save and create functions are not used for now, let's remove them

Reply:
https://github.com/simeontg/cinema-frontend/pull/3/commits/d878e8bb84630191bf981962a5198f44d4627aaa#diff-f6af47277e80e3559a7ceb928bd6521ba267ee3bc8eba048a4314e3fa8726e2f